pcsx2-qt/Settings/GraphicsSettingsWidget.cpp

```diff
@@ -614,7 +614,7 @@ GraphicsSettingsWidget::GraphicsSettingsWidget(SettingsDialog* dialog, QWidget*
 			tr("Shows on-screen-display messages when events occur such as save states being "
 			   "created/loaded, screenshots being taken, etc."));
 
-		dialog->registerWidgetHelp(m_ui.osdShowFPS, tr("Show Game Frame Rate"), tr("Unchecked"),
+		dialog->registerWidgetHelp(m_ui.osdShowFPS, tr("Show Internal FPS"), tr("Unchecked"),
```

Comment:
Leave it as Game Frame Rate, this is done on purpose because users don't know what "internal" means.

Reply:
![Sin título](https://user-images.githubusercontent.com/5641681/221405188-3140e358-e8a9-4959-a217-572b2e3f5e41.jpg)

Reply:
Leave `Show FPS` as is and adjust this one to match

Reply:
> Leave `Show FPS` as is and adjust this one to match

Done.